viewer/packages/utilities/src/events/EventTrigger.ts

```diff
@@ -2,10 +2,25 @@
  * Copyright 2021 Cognite AS
  */
 
+/**
+ * A view into an event viewer trigger
+ */
+export interface EventTriggerView<TListener extends (...args: any[]) => void> {
+  /**
+   * Get all event listeners subscribed to this trigger currently
+   */
+  getListeners(): TListener[];
+
+  /**
+   * Replace all currently registered listeners
+   */
+  replaceListeners(listeners: TListener[]): TListener[];
```

Comment:
API seems a little strange for me as mentioned in the comment for `MeasurementTool.ts` file